src/main/java/stelnet/ui/Button.java

```diff
@@ -9,93 +9,72 @@
 import com.fs.starfarer.api.ui.TooltipMakerAPI;
 import com.fs.starfarer.api.util.Misc;
 
-public class Button extends Renderable implements Callable {
+import lombok.Getter;
+import lombok.Setter;
 
-    private final Size size;
-    private final String title;
+public class Button extends Renderable implements ButtonHandler {
```

Comment:
Ditch the `@Getter @Setter` and use

```
@Data
public class Button extends Renderable implements ButtonHandler {
```
If you really really do not want to expose title and handler via setters then you can:

```
@Setter(value = AccessLevel.NONE)
private int mySecret;
```